docs/_blog/miniscript_vulnerability.md

```diff
@@ -0,0 +1,68 @@
+---
+title: "Security Advisory: Miniscript MINIMALIF Bug"
+description: "Security advisory on the recent Miniscript MINIMALIF bug. How to check if you are affected and what to do next."
+authors:
+    - Alekos Filini
+date: "2022-04-19"
+tags: ["miniscript", "security"]
+hidden: true
+draft: false
+---
+
+BDK relies heavily on the Miniscript language to support arbitrary output descriptors seamlessly.
+
+The Miniscript compiler models arbitrary spending policies and generates an optimized bitcoin script that enforces said policies when executed in the bitcoin network.
+
+Recently a bug has been discovered in the Miniscript type system, which can cause an unsafe bitcoin script to be generated in some cases, described in detail below. Affected UTXOs can be spent by malicious
+miners without providing any valid signature, effectively bypassing all the signature checks in the script.
+
+We analyzed mainnet blocks starting from the beginning of 2020 and **found no transaction spending from a script which is affected by this bug**, which leads us to believe that nobody is currenly using this
+structure in production. Nonetheless, we still recommend upgrading as soon as possible.
+
+## How to check if you are vulnerable
+
+As a rule of thumb, if your descriptor contains a `thresh()` with either an `older()` or `after()` inside, then you are probably vulnerable.
```

Comment:
It is also important to note that funds are only vulnerable if timelock conditions can be met. If the timelock conditions have not been met and the funds can be moved, it will be safe to move the funds.